src/common/box_filters.c

```diff
@@ -226,13 +241,17 @@ static void sub_4wide_Kahan(float *const restrict accum, const dt_aligned_pixel_
   }
 }
 
-static void store_scaled_4wide(float *const restrict out, const dt_aligned_pixel_t in, const float scale)
+static void store_scaled_4wide(
+    float *const restrict out,
+    const dt_aligned_pixel_t in,
+    const float scale)
 {
   for_four_channels(c,aligned(in))
     out[c] = in[c] / scale;
 }
 
-static void sub_16wide(float *const restrict accum, const float *const restrict values)
+static void _sub_16wide(float *const restrict accum,
+                       const float *const restrict values)
```

Comment:
Indent.

BTW, this indentation is ok but the one above not. The style is really to have the first parameter on the same line as the routine name. Not a blocking point for merging this, but you can fix that in this PR if you're motivated :) But at least keep that in mind for the next PRs.

Reply:
Got it.